### PR TITLE
Add some developper documentation

### PR DESCRIPTION
**Pull request by frost242**
Two developer-specific docs are now available : 

- releasing.md
- contributing.md

The first is intended for the mainteners. The second for newcomers to the project that want to contribute to the project.

A last file lists the contributors to the project.

### CODE REVIEW

**Comment by frost242** [comment]
Requested changes done. Please let me know.

File: docs/dev/contributing.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+```
+
+Several other variables are defined :
+
+* @rs: array to store the result set of the monitoring query
+* @perfdata: array to store the returned perdata
+* @msg: array to store the returned messages
+* @hosts: array to know the host(s) to query
+* %args
+
```
Comment (ioguix) [inline docs/dev/contributing.md]: markdown-escape these variables

File: docs/dev/contributing.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+Consider using these variables names as a convention.
+
+Also populate your own %args hash from the first argument :
```
Comment (ioguix) [inline docs/dev/contributing.md]: escape

**Comment by frost242** [comment]
The perfdata array is explained a little later, see contributing.md from line 113.

Done for the rest.

**Pull request merged**